media-tv/tvbrowser-bin/tvbrowser-bin-4.0.1.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit desktop
+
+DESCRIPTION="Themeable and easy to use TV Guide - written in Java"
+HOMEPAGE="https://www.tvbrowser.org"
+MY_PN="tvbrowser"
+MY_P="${MY_PN}-${PV}"
+SRC_URI="mirror://sourceforge/project/tvbrowser/TV-Browser%20Releases%20%28Java%208%20and%20higher%29/${PV}/${MY_PN}_${PV}_bin.tar.gz"
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
```

Comment:
One thing that came to my mind: Since it's a precompiled binary, does it work on amd64 and x86? Have you tested it on both arches? :)

Reply:
I have not tested it on x86, but it is written in java and therefore should be platform-independent. At least it states to be on the homepage. It should probably work on other arches where the java runtime can run.